src/components/AvatarWithImagePicker.js

```diff
@@ -21,6 +19,7 @@ import SpinningIndicatorAnimation from '../styles/animation/SpinningIndicatorAni
 import Tooltip from './Tooltip';
 import stylePropTypes from '../styles/stylePropTypes';
 import * as FileUtils from '../libs/fileDownload/FileUtils';
+import getImageResolution from '../libs/fileDownload/FileUtils';
```

Comment:
'../libs/fileDownload/FileUtils' does not have getImageResolution function.

Reply:
```suggestion
import getImageResolution from '../libs/fileDownload/getImageResolution';
```

Reply:
fixed